scripts/updates/github-projects.txt

```diff
@@ -181,6 +181,7 @@ gumbo-parser|google/gumbo-parser
 harfbuzz|harfbuzz/harfbuzz
 hashdeep|jessek/hashdeep
 helm|helm/helm
+helix|helix-editor/helix
```

Comment:
Need to move it up one more, i before m.

Reply:
😅 No word for such nonsense mistake!